### PR TITLE
Update extend -> extension

### PR DESCRIPTION
**Pull request by elanalynn**
### Background

Many templates were using old `extend` method to register extensions. 

### Solution

Update to current method for registering extensions. 

`extend` -> `extension`

### Checklist

- [x] I have :tophat:'d these changes
- [x] I have squashed my commits into chunks of work with meaningful commit messages

### CODE REVIEW

File: checkout-post-purchase/src/index.liquid [hunk 1]
```diff
@@ -133,7 +133,7 @@ import {
  * optionally allows data to be stored on the client for use in the `Render`
  * extension point.
  */
- extend("Checkout::PostPurchase::ShouldRender", async ({ storage }) => {
+extend("Checkout::PostPurchase::ShouldRender", async ({ storage }) => {
```
Comment (vividviolet) [inline checkout-post-purchase/src/index.liquid:136]: Let's revert this change to remove the space

Comment (elanalynn) [reply checkout-post-purchase/src/index.liquid:136]: @vividviolet  I saw this as well, but I think the space was off before. It wasn't aligned with the functions around it.

Comment (vividviolet) [reply checkout-post-purchase/src/index.liquid:136]: Oops, you're right. Ignore me
